fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2ListSequencesForSchema.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.db2;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.ibm.fhir.database.utils.api.IDatabaseSupplier;
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+import com.ibm.fhir.database.utils.common.DataDefinitionUtil;
+import com.ibm.fhir.database.utils.common.SchemaInfoObject;
+import com.ibm.fhir.database.utils.common.SchemaInfoObject.Type;
+
+/**
+ * DAO to fetch the names of sequences in the given schema
+ */
+public class Db2ListSequencesForSchema implements IDatabaseSupplier<List<SchemaInfoObject>> {
```

Comment:
ListTablesDAO?

Reply:
Per above, never used polymorphically so I didn't see the point in the end